hack/configapi/main.go

```diff
@@ -42,20 +46,18 @@ func handleError(err error) {
 
 func main() {
 	myCmd := &cobra.Command{
-		Use:   "upload a set of versions specific to the azure-sev-snp attestation variant to the config api",
+		Use:   fmt.Sprintf("please set the %s and %s environment variables and all flags", envAwsKeyID, envAwsKey),
 		Short: "upload a set of versions specific to the azure-sev-snp attestation variant to the config api",
 		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
```suggestion
		RunE: func(cmd *cobra.Command, args []string) error {
```
Use `RunE` to offload error handling to caller of the command.
You can then remove all calls to `panic` inside the command`. (Also helps if we want to add unit tests at some point)